pkgs/servers/x11/xarcan/default.nix

```diff
@@ -0,0 +1,91 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, meson
+, pkg-config
+, xorgproto
+, xtrans
+, libxshmfence
+, pixman
+, xkbcomp
+, libxkbfile
+, libXfont2
+, dbus
+, systemd
+, fontutil
+, mesa
+, epoxy
+, libmd
+, nettle
+, libgcrypt
+, openssl
+, libXdmcp
+, libdrm
+, libselinux
+, audit
+, libGL
+, libtirpc
+, libXau
+, arcan
+, libxcb
+, ninja
+, libX11
+}:
+
+stdenv.mkDerivation rec {
+  pname = "xarcan";
+
+  version = "0.6.0+unstable=2021-05-27";
+
+  src = fetchFromGitHub {
+    owner = "letoram";
+    repo = pname;
+    rev = "3dc5f8114bef88104e93276c2ccd957c0789df3a";
+    sha256 = "sha256-zlk3Jw3YVW59AvWwD8qZJTujPgsKSxUK068bgpkhHl0=";
+  };
+
+  nativeBuildInputs = [
+    meson
+    pkg-config
+  ];
+
+  buildInputs = [
+    xorgproto
+    xtrans
+    libxshmfence
+    pixman
+    xkbcomp
+    libxkbfile
+    libXfont2
+    dbus
+    systemd
+    fontutil
+    mesa
+    epoxy
+    libmd
+    nettle
+    libgcrypt
+    openssl
+    libXdmcp
+    libdrm
+    libselinux
+    audit
+    libGL
+    libtirpc
+    libXau
+    arcan
+    libxcb
+    ninja
+    libX11
+  ];
+
+  meta = with lib; {
+    description = "Patched Xserver that bridges connections to Arcan";
+    longDescription = "";
```

Comment:
```suggestion
```

Reply:
Why shouldn't the application have a long description?

Reply:
You set an empty one. Either add content to it or remove it